actionview/test/template/resolver_shared_tests.rb

```diff
@@ -82,4 +86,30 @@ def test_doesnt_find_template_with_wrong_details
     templates = resolver.find_all("hello_world", "test", false, locale: [], formats: [:xml], variants: :any, handlers: [:erb])
     assert_equal 0, templates.size
   end
+
+  def test_found_template_is_cached
+    with_file "test/hello_world.html.erb", "Hello HTML!"
+
+    a = context.find("hello_world", "test", false, [], {})
+    b = context.find("hello_world", "test", false, [], {})
+    assert_same a, b
+  end
+
+  def test_same_template_from_different_details_is_same_object
+    with_file "test/hello_world.html.erb", "Hello plain text!"
```

Comment:
Do we have a test for the `test/hellow_world.fr.html.rb`?

Reply:
👍 Added an additional test in 2455d16 to show that:
1. Only fr gets the `.fr.html.erb` template
2. If there are multiple results, they are still de-duped